functional_test.sh

```diff
@@ -35,23 +39,36 @@ docker run \
     -classpath web/target/graphhopper-web-1.0-SNAPSHOT.jar com.graphhopper.http.GraphHopperApplication gtfs_links default_gh_config.yaml
 
 # Run server in background
-docker run --rm  --name functional_test_server -p 50051:50051 -p 8998:8998 -v "$TMPDIR:/graphhopper/transit_data/" \
+docker run --rm --name functional_test_server -p 50051:50051 -p 8998:8998 -v "$TMPDIR:/graphhopper/transit_data/" \
     "$DOCKER_IMAGE_TAG" &
 
 echo "Waiting for graphhopper server to start up"
-sleep 30
+sleep 30 # should we implement a trie mechanism with time threshold instead of sleep?
```

Comment:
To answer the question in the comment, I think we decided that we didn't want to spend development time on that, since we didn't think there was that much to gain (and this shouldn't run that often)